OctoAwesome/OctoAwesome/OrientationFlags.cs

```diff
@@ -118,62 +118,62 @@ public enum OrientationFlags : byte
         #region Edges
 
         /// <summary>
-        /// Beschreibt die obere Kante der Westseite [-X,-Y,+Z] -> [-X,+Y,+Z]
+        /// Describes the top edge on the west-side [-X,-Y,+Z] -> [-X,+Y,+Z]
         /// </summary>
         EdgeWestTop = Corner001 | Corner011,
 
         /// <summary>
-        /// Beschreibt die untere Kante der Westseite [-X,-Y,-Z] -> [-X,+Y,-Z]
+        /// Describes the bottom edge on the west-side [-X,-Y,-Z] -> [-X,+Y,-Z]
         /// </summary>
         EdgeWestBottom = Corner000 | Corner010,
 
         /// <summary>
-        /// Beschreibt die obere Kante der Ostseite [+X,-Y,+Z] -> [+X,+Y,+Z]
+        /// Describes the top edge on the east-side [+X,-Y,+Z] -> [+X,+Y,+Z]
         /// </summary>
         EdgeEastTop = Corner101 | Corner111,
 
         /// <summary>
-        /// Beschreibt die untere Kante der Ostseite [+X,-Y,-Z] -> [+X,+Y,-Z]
+        /// Describes the bottom edge on the east-side [+X,-Y,-Z] -> [+X,+Y,-Z]
         /// </summary>
         EdgeEastBottom = Corner100 | Corner110,
 
         /// <summary>
-        /// Beschreibt die obere Kante der Nordseite [-X,+Y,+Z] -> [+X,+Y,+Z]
+        /// Describes the top edge on the north-side [-X,+Y,+Z] -> [+X,+Y,+Z]
         /// </summary>
         EdgeNorthTop = Corner011 | Corner111,
 
         /// <summary>
-        /// Beschreibt die untere Kante der Nordseite [-X,+Y,-Z] -> [+X,+Y,-Z]
+        /// Describes the bottom edge on the north-side [-X,+Y,-Z] -> [+X,+Y,-Z]
         /// </summary>
         EdgeNorthBottom = Corner010 | Corner110,
 
         /// <summary>
-        /// Beschreibt die obere Kante der Südseite [-X,-Y,+Z] -> [+X,-Y,+Z]
+        /// Describes the top edge on the south-side [-X,-Y,+Z] -> [+X,-Y,+Z]
         /// </summary>
         EdgeSouthTop = Corner001 | Corner101,
 
         /// <summary>
-        /// Beschreibt die untere Kante der Südseite [-X,-Y,-Z] -> [+X,-Y,-Z]
+        /// Describes the bottom edge on the south-side [-X,-Y,-Z] -> [+X,-Y,-Z]
         /// </summary>
         EdgeSouthBottom = Corner000 | Corner100,
 
         /// <summary>
-        /// Beschreibt die Senkrechte zwischen Nord und West [-X,+Y,-Z] -> [-X,+Y,+Z]
+        /// Describes the upright edge between north and west [-X,+Y,-Z] -> [-X,+Y,+Z]
         /// </summary>
         EdgeNorthWest = Corner010 | Corner011,
 
         /// <summary>
-        /// Beschreibt die Senkrechte zwischen Nord und Ost [+X,+Y,-Z] -> [+X,+Y,+Z]
+        /// Describes the upright edge between north and east [+X,+Y,-Z] -> [+X,+Y,+Z]
         /// </summary>
         EdgeNorthEast = Corner110 | Corner111,
 
         /// <summary>
-        /// Beschreibt die Senkrechte zwischen Süd und West [-X,-Y,-Z] -> [-X,-Y,+Z]
+        /// Describes the upright edge between Süd and west [-X,-Y,-Z] -> [-X,-Y,+Z]
```

Comment:
```suggestion
        /// Describes the upright edge between south and west [-X,-Y,-Z] -> [-X,-Y,+Z]
```

I didn't know the english alphabet included the letter "ü"